en/includes/guides/webhooks/setup-webhooks.md

```diff
@@ -0,0 +1,255 @@
+/hub.mode=subscribe&hub.topic=myorg.dcbd6baa-40c7-4eb9-9c2e-19b868d0e266.schema.wso2.v1.event.login&hub.challenge=500c4833-91c6-4556-b9a9-1b8e5e00d48b&hub.lease_seconds=864000
+
+```
+
+In this request:
+
+- <code>hub.mode=subscribe</code> indicates that {{product_name}} is verifying a subscription.
+- <code>hub.topic</code> shows the URL of the topic you subscribe to.
+
+    {{product_name}} creates the topic in the following format:
+
+    *{your-root-organization-domain}.{root-or-sub-organization-id}.schema.{subscribed-event-profile-name}.{subscribed-event-profile-version}.event.{subscribed-event-type-name}*
+
+- <code>hub.challenge</code> is a unique, random string generated by {{product_name}}. Your endpoint must return this exact string in its response body.
+- <code>hub.lease_seconds</code> specifies how long the subscription will remain active.
+
+Upon receiving this request, your endpoint must respond with a <code>2xx</code> HTTP status code and include the exact <code>hub.challenge</code> value in the response body.
+
+**Example response**:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: text/plain
+
+500c4833-91c6-4556-b9a9-1b8e5e00d48b
+```
+
+This process adheres to the [WebSub specification](https://www.w3.org/TR/2018/REC-websub-20180123/#hub-verifies-intent), which defines how hubs verify subscription intent.
+
+## Receive webhook events
+
+{{product_name}} sends an HTTP POST request to your webhook endpoint when a subscribed event is triggered. This request contains a JSON message with event details. Your service can then parse and use this information as needed.
+
+### Event payload structure
+
+Webhook event payloads adhere to the [Security Event Token (SET) specification (RFC 8417)](https://www.rfc-editor.org/rfc/rfc8417.html). This ensures a standardized and secure format for event information. Each payload includes:
+
+- <code>iss</code> **(Issuer)**: Identifies the {{product_name}} instance that issued the event.  
+- <code>iat</code> **(Issued At)**: Timestamp showing when {{product_name}} issues the event.  
+- <code>jti</code> **(JWT ID)**: A unique identifier for the event.  
+- <code>events</code>: This object holds the actual event data. Its internal structure changes depending on the specific event types you've subscribed to. For instance, subscribing to **Logins** will deliver both successful and failed login events, with distinct data structures for each.
+
+**Example event payload for login success event**:
+
+```http
+POST /your-webhook-endpoint
+Content-Type: application/json
+X-Hub-Signature: sha256=abcdef12345... (HMAC signature)
+
+{
+  "iss": "https://api.asgardeo.io/t/myorg",
+  "jti": "051f0c37-b689-44d4-b7d2-29b980ece273",
+  "iat": 1751705149662,
+  "events": {
+    "https://schemas.identity.wso2.org/events/login/event-type/loginSuccess": {
+      "user": {
+        "id": "d4002616-f00c-49d5-b9b7-63b063819049",
+        "ref": "https://api.asgardeo.io/t/myorg/scim2/Users/d4002616-f00c-49d5-b9b7-63b063819049"
```

Comment:
Will this file be included in to IS docs as well ?
If so, better to template the hostname and override by the product specific docs?

eg: hostname of iss and ref of the user